### PR TITLE
Components

### PR DESCRIPTION
**Pull request by tawfiknasser**
- changed App to a react class
- added styles to the components
- added card, header, search components

### CODE REVIEW

**Comment by shiryz** [comment]
@Ryan2601 @yoosefkh assign me again please once you've reviewed

File: client/src/App.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
-import React from 'react'
-const App = () => {
+
+import React, { Component } from 'react';
+import SearchRectangle from './components/search_info_components/search';
```
Comment (shiryz) [inline client/src/App.js]: Can you change SearchRectangle to something more meaningful? SearchBar or something similar would be better

File: client/src/App.js [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+    <React.Fragment>
+    <HeaderBar />
+    <SearchRectangle/>
+    <InfoBar />
+    </React.Fragment>
 
-    </div>
   );
-};
+}
```
Comment (shiryz) [inline client/src/App.js]: Why aren't using a linter or prettier?

File: client/src/components/Header_Bar/header.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+.HeadRectangle {
```
Comment (shiryz) [inline client/src/components/Header_Bar/header.css]: Please stay away from naming something *Rectangle, use things like box or container for example

File: client/src/components/Header_Bar/header.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  position: absolute;
+  width: 375px;
+  height: 57px;
+  left: 0px;
+  top: 0px;
+
+  background: #ffca28;
+}
+
+.textHeader {
```
Comment (shiryz) [inline client/src/components/Header_Bar/header.css]: Decide which naming convention you want to use and apply that to all, you've mixed two in here (HeadRectangle, textHeader...)

**Comment by shiryz** [comment]
You also need to reference the issue in your description

**Comment by MynahMarie** [comment]
If @shiriz is happy with this one then I'm happy too, we can merge.

**Pull request merged**